cmd/mimir/help-all.txt.tmpl

```diff
@@ -1325,6 +1329,8 @@ Usage of ./cmd/mimir/mimir:
     	Stream chunks from ingesters to queriers. (default true)
   -ingester.tsdb-config-update-period duration
     	[experimental] Period with which to update the per-tenant TSDB configuration. (default 15s)
+  -ingester.utilization-based-limiting-enabled
```

Comment:
Perhaps just `utilization-limits-enabled`?

Reply:
It is simpler, making the change.